### PR TITLE
create table bats

### PR DESCRIPTION
**Pull request by andrew-wm-arthur**

### CODE REVIEW

File: bats/create-tables.bats [hunk 1]
```diff
@@ -202,18 +199,23 @@ teardown() {
...
+    # [[ "$output" =~ "unsupported characters" ]] || false
+
     run dolt table import -c --pk=pk test `batshelper bad-characters.csv`
-    skip "Dolt allows you to create tables with non-UTF-8 characters right now"
-    [ "$status" -eq 1 ]
-    [[ "$output" =~ "unsupported characters" ]] || false
+    [ "$status" -eq 0 ]
+    dolt sql -q 'select * from test'
+    dolt sql -r csv -q 'select * from test' > compare.csv
+    diff compare.csv `batshelper bad-characters.csv`
```
Comment (andrew-wm-arthur) [inline bats/create-tables.bats:218]: I changed this test to assert that we store/retrieve non-UTF8 characters correctly. 

We can validate that strings are UTF8 and error, but it's even easier to just support everything

File: bats/helper/bad-characters.csv [hunk 2]
```diff
@@ -1,2 +1,2 @@
 pk,c1
-0,�
\ No newline at end of file
+0,�
```
Comment (andrew-wm-arthur) [inline bats/helper/bad-characters.csv:2]: added newline to match the output of `dolt sql -r csv -q 'select * from test'`

File: bats/create-tables.bats [hunk 3]
```diff
@@ -55,23 +55,20 @@ teardown() {
 }
 
 @test "create a table with json import. bad json." {
```
Comment (bheni) [inline bats/create-tables.bats:57]: change the test name to be accurate

File: bats/create-tables.bats [hunk 4]
```diff
@@ -55,23 +55,20 @@ teardown() {
...
 @test "create a table with json import. bad schema." {
-    run dolt table import -c -s `nativebatsdir employees-sch-bad.json` employees `batshelper employees-tbl.json`
+    run dolt table import -c -s `batshelper employees-sch-bad.sql` employees `batshelper employees-tbl.json`
     [ "$status" -eq 1 ]
     [[ "$output" =~ "Error determining the output schema" ]] || false
-    skip "Error message mentions valid table file but not invalid schema file"
-    # Be careful here. "employees-sch-bad.json" matches. I think it is because
-    # the command line is somehow in $output. Added " to" to make it fail.
-    [[ "$output" =~ "employees-sch-bad.json to" ]] || false
+    [[ "$output" =~ "syntax error at position 32 near 'And'" ]] || false
```
Comment (bheni) [inline bats/create-tables.bats:70]: I don't think it's necessary to test the full output message.  The first line is probably enough.  It makes it more fragile.  If I change the output error message I gotta update every test that counts on it returning an exact message.

File: bats/helper/employees-sch-bad.sql [hunk 5]
```diff
@@ -0,0 +1,3 @@
+CREATE TABLE employees (
+   And then I said, supposing you brought the light inside of the body... Sounds interesting.
```
Comment (bheni) [inline bats/helper/employees-sch-bad.sql:2]: hahahaha

**Pull request merged**